src/nvim/ex_cmds2.c

```diff
@@ -3324,7 +3324,7 @@ static char_u **find_locales(void)
 {
   garray_T locales_ga;
   char_u      *loc;
-  char *strtok_r_save_ptr = NULL;
+  char *saveptr = NULL;
 
   // Find all available locales by running command "locale -a".  If this
   // doesn't work we won't have completion. */
```

Comment:
Thanks for pointing it out, I must have missed some. I'll go through the whole diff again and remove these and any thing else I have missed.